android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/GpsFactory.java

```diff
@@ -66,14 +66,15 @@ public static ValidationStatus validate(JsonFormFragmentView formFragmentView,
 
     public static String constructString(Location location) {
         if (location != null) {
-            return constructString(String.valueOf(location.getLatitude()), String.valueOf(location.getLongitude()));
+            return constructString(String.valueOf(location.getLatitude()), String.valueOf(location.getLongitude()),
+                    String.valueOf(location.getAltitude()), String.valueOf(location.getAccuracy()));
```

Comment:
Consider creating a method that takes a collection of properties and iterates over them calling `String.valueOf` since it's repeated multiple times here.